src/main/java/edu/hm/hafner/analysis/parser/NagFortranParser.java

```diff
@@ -41,6 +44,27 @@ public NagFortranParser() {
                 .buildOptional();
     }
 
+    @Override
+    protected Optional<Issue> createIssue(final Matcher matcher, final LookaheadStream lookahead,
+            final IssueBuilder builder)
+            throws ParsingException {
+
+            StringBuilder messageBuilder = new StringBuilder(matcher.group(5));
+            String category = matcher.group(1);
+
+            while (lookahead.hasNext("\\s+ .+")) {
+                messageBuilder.append("\n");
+                messageBuilder.append(lookahead.next());
+            }
+
+            return builder.setFileName(matcher.group(2))
+                    .setLineStart(getLineNumber(matcher))
```

Comment:
replace with `parseInt(matcher.group(4))` and remove `getLineNumber` method

Reply:
The test data contains entries that do not have a line number (`NagFortranError.txt`), so `parseInt` would throw an `NumberFormatException`. Is the test data valid or must it be modified?

Reply:
Ah: `IntegerParser.parseInt` does handle the exception already, it is used by all other parsers as well.